Package.swift

```diff
@@ -13,17 +13,29 @@ let package = Package(
         .visionOS("1.0"),
     ],
     products: [
-        // Products define the executables and libraries a package produces, making them visible to other packages.
         .library(
             name: "ToonFormat",
             targets: ["ToonFormat"]
-        )
+        ),
+        .library(
+            name: "TOONEncoder",
+            targets: ["TOONEncoder"]
+        ),
+        .library(
+            name: "TOONDecoder",
+            targets: ["TOONDecoder"]
+        ),
```

Comment:
I don't think the benefits of splitting these up into separate modules is worth the trade-off of complexity this introduces. Even assuming that 99% of API consumers never use `TOONDecoder`, the impact of these unused code symbols is minimal.